### DISCUSSION TITLE
How do I disable the reading time field correctly?

### DISCUSSION THREAD

**Discussion by hydraumech** [vuepress-theme-hope/vuepress-theme-hope#3147] [Q&A] [answered]
### 
-
sidebar: false
pageview: false
ArticleInfo["ReadingTime"] | false // I know this is not correct.
-

**Comment by Mister-Hope** [comment, marked as answer]
You need to read the docs and at least understand some js, markdown and yaml basics, that shouldn't be hard, the docs are already detailed enough.

Readingtime can be globally disabled, and you can globally config pageview items with `pageview` in theme options, and you can also do that in frontmatter to change that in one page locally.
